src/rmats2sashimiplot/rmats2sashimiplot.py

```diff
@@ -6,32 +6,41 @@
 import subprocess
 
 
-def convert_sam2bam(options):
-    """
-    convert sam files to bam files and store the filename in options.b1 & options.b2
-    """
-    if options.s1:  # input with sam file
-        sample_1 = options.s1.split(",")
-        for sam in sample_1:
-            os.system("samtools view -Sbh " + sam + " > " + sam.replace(".sam", ".bam"))
-        options.b1 = options.s1.replace(".sam", ".bam")
-        sample_2 = options.s2.split(",")
-        for sam in sample_2:
-            os.system("samtools view -Sbh " + sam + " > " + sam.replace(".sam", ".bam"))
-        options.b2 = options.s2.replace(".sam", ".bam")
-    for bam in options.b1.split(","):
+def convert_sams2bams(sams):
+    separator = ","
+    split_sams = sams.split(separator)
+    bams = list()
+    for sam in split_sams:
+        bam = sam.replace(".sam", ".bam")
+        bams.append(bam)
+        os.system("samtools view -Sbh {} > {}".format(sam, bam))
+
+    return separator.join(bams)
+
+
+def index_bams(bams):
+    split_bams = bams.split(",")
+    for bam in split_bams:
         if os.path.isfile(bam + '.bai'):  # if the bam file has been indexed.
             print("'{0}' is indexed already: '{0}.bai'".format(bam))
         else:
             print("Indexing '{}'.".format(bam))
             os.system("samtools index " + bam)
-    for bam in options.b2.split(","):
-        if os.path.isfile(bam + '.bai'):
-            print("'{0}' is indexed already: '{0}.bai'".format(bam))
-        else:
-            print("Indexing '{}'.".format(bam))
-            os.system("samtools index " + bam)
-    return
+
+
+def prepare_bams(options):
```

Comment:
For cases that users only provide s2/b2 without s1/b1. The indent of the if statement '`if options.s2:`' should be decreased. Also, there should be another if statement for the  'options.b1'.